src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/canceltranslations/CancelTranslationsAboutToSubmitHandlerTest.java

```diff
@@ -59,120 +61,128 @@ public void givenCanHandleIsCalledWithValidCallBack_shouldReturnTrue() {
-                        DocumentType.REINSTATEMENT_REQUEST
-                                .getValue(), LocalDate.now().minusDays(2));
-
-        List<SscsDocument> docs = Arrays.asList(ssc0Doc, sscs1Doc, sscs2Doc, sscs3Doc, sscs4Doc, sscs5Doc, sscs6Doc);
+    private Callback<SscsCaseData> buildCallback(EventType eventType, State state, int sscsDocumentNumber) {
+        SscsDocument sscsDocument = switch (sscsDocumentNumber) {
+            case 0 -> buildSscsDocument("english.pdf", SscsDocumentTranslationStatus.TRANSLATION_REQUIRED,
+                DocumentType.APPELLANT_EVIDENCE.getValue(), LocalDate.now());
+            case 1 -> buildSscsDocument("english.pdf", SscsDocumentTranslationStatus.TRANSLATION_REQUIRED,
+                DocumentType.SSCS1.getValue(), LocalDate.now().minusDays(1));
+            case 2 -> buildSscsDocument("anything.pdf", SscsDocumentTranslationStatus.TRANSLATION_REQUIRED,
+                DocumentType.DECISION_NOTICE
+                    .getValue(), LocalDate.now().minusDays(3));
+            case 3 -> buildSscsDocument("anything.pdf", SscsDocumentTranslationStatus.TRANSLATION_COMPLETE,
+                DocumentType.DIRECTION_NOTICE
+                    .getValue(), LocalDate.now().minusDays(2));
+            case 4 -> buildSscsDocument("anything.pdf", null, DocumentType.DECISION_NOTICE
+                .getValue(), LocalDate.now().minusDays(2));
+            case 5 -> buildSscsDocument("anything.pdf", SscsDocumentTranslationStatus.TRANSLATION_REQUESTED,
+                DocumentType.DIRECTION_NOTICE
+                    .getValue(), LocalDate.now().minusDays(2));
+            case 6 -> buildSscsDocument("anything.pdf", SscsDocumentTranslationStatus.TRANSLATION_REQUIRED,
+                DocumentType.REINSTATEMENT_REQUEST
+                    .getValue(), LocalDate.now().minusDays(2));
+            case 8 -> buildSscsDocument("ibc.pdf", SscsDocumentTranslationStatus.TRANSLATION_REQUIRED,
+                DocumentType.SSCS8.getValue(), LocalDate.now().minusDays(4));
+            default -> null;
+        };
 
         SscsCaseData sscsCaseData = SscsCaseData.builder()
-                .sscsDocument(docs)
-                .state(State.VALID_APPEAL)
-                .languagePreferenceWelsh("Yes")
-                .build();
+            .sscsDocument(Collections.singletonList(sscsDocument))
+            .state(State.VALID_APPEAL)
+            .languagePreferenceWelsh("Yes")
+            .build();
         CaseDetails<SscsCaseData> caseDetails = new CaseDetails<>(123L, "sscs",
-                state, sscsCaseData, LocalDateTime.now(), "Benefit");
+            state, sscsCaseData, LocalDateTime.now(), "Benefit");
         return new Callback<>(caseDetails, Optional.empty(), eventType, false);
     }
 
     private SscsDocument buildSscsDocument(String s, SscsDocumentTranslationStatus translationRequired, String docType,
```

Comment:
could we rename the first method argument to filename

Reply:
actually, given that the filename isn't being asserted, we could just hardcode it, so it's one less argument